.gitignore

```diff
@@ -167,3 +167,5 @@ msbuild.binlog
 BenchmarkDotNet.Artifacts
 
 artifacts
+
+Azure.Core.All.sln
```

Comment:
This would actually ignore any changes to the one in sdk/core as well. Maybe better to delete the file and fix the script so that it's always in the same place? /cc @pakrym

Reply:
deferring to https://github.com/Azure/azure-sdk-for-net/issues/17291